src/analysis/functions.rs

```diff
@@ -389,7 +397,7 @@ fn analyze_function(
             Some(obj) => {
                 match env.library.type_(type_tid) {
                     library::Type::Class(_) => obj.concurrency,
-                    _ => library::Concurrency::SendSync,
+                    _ => library::Concurrency::None,
```

Comment:
Maybe interface and record need be here too,
please add comment for checking later (at first glance I don't find any)

Reply:
Good point! I add the comment.

Reply:
Both are needed here, yes.

Reply:
I mean, add them and not just a FIXME comment :P